src/components/dropdown-filter-ajax/dropdown-filter-ajax.js

```diff
@@ -198,6 +222,14 @@ class DropdownFilterAjax extends DropdownFilter {
         filter: this.props.create ? prevState.filter : null
       }));
 
+      if (this.getDataTimeout) {
```

Comment:
This sounds a bit confusing, getDataTimeout sounds like a getter method rather than class member - `dataFetchTimeoutId` would sound quite a bit more like a variable that holds timeout id not an action? or something shorter but not starting with a verb.